recipe/LICENSE

```diff
@@ -1,5 +1,5 @@
 BSD 3-clause license
-Copyright (c) 2015-2019, conda-forge
+Copyright (c) 2015-2020, conda-forge contributors
 All rights reserved.
 
 Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
```

Comment:
Why was this file moved to the `recipe` directory?

For the context: there are 2 license files in this repo. One file resides in the root of the repo, and is related to the infrastructure files provided by conda-smithy. Another file (optional) can reside in the `recipe` directory, and corresponds to the package's license, i.e. to the `xpdacq` in this case: https://github.com/xpdAcq/xpdAcq/blob/master/LICENSE. If the file is provided by the GitHub or PyPI sources, the file in the `recipe` directory can be omitted, but the `about: license_file: LICENSE` entry of the `meta.yaml` file should still list it.

I recommend to revert this change.

Reply:
I made a mistake. I thought the file that I moved was the package license. I will revert the change.

Reply:
Thanks.